Makefile

```diff
@@ -164,8 +164,8 @@ help:
 	@echo " - revive            run revive linter"
 	@echo " - misspell          check for misspellings"
 	@echo " - vet               examines Go source code and reports suspicious constructs"
-	@echo " - test              run unit test"
-	@echo " - test-sqlite       run integration test for sqlite"
+	@echo " - test[\#TestSpecificName]    			run unit test"
+	@echo " - test-sqlite[\#TestSpecificName]       run integration test for sqlite"
 	@echo " - pr#<index>        build and start gitea from a PR with integration test data loaded"
```

Comment:
I think you have to add space for the oter entrys too to have the same aligment

Reply:
I wander whether It's a good style ...🤔

![image](https://user-images.githubusercontent.com/25342410/81695772-a48d8f00-9495-11ea-82a1-c50dd6850041.png)

Reply:
depending on the window size ...